frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -117,11 +117,12 @@ const DevPortfolioForm: React.FC = () => {
         status: 'pending',
         ...(text && { text })
       };
-      sendSubmissionRequest(newDevPortfolioSubmission, devPortfolio);
-      setDevPortfolio(undefined);
-      setOpenPRs(['']);
-      setReviewedPRs(['']);
-      setText(null);
+      sendSubmissionRequest(newDevPortfolioSubmission, devPortfolio).then(() => {
+        setDevPortfolio(undefined);
+        setOpenPRs(['']);
+        setReviewedPRs(['']);
+        setText('');
```

Comment:
For some reason setting text to `undefined` or `null` doesn't clear the `TextArea` 🤷🏻‍♂️